backend/appwrite-functions/announcement/README.md

```diff
@@ -0,0 +1,102 @@
+# General
+This function brings add and get announcements features to appwrite.
+- Client can send list of announcements to Backend. These data will be written to DB along with `created_time`
+- Client can request annoucements with some filters. The function will then query data from DB and print out to `stdout`.
+    Client can now request status of execution and retrieve data from the `stdout` or `stderr` field. See `backend/appwrite-functions/announcement/test.js`.
+- `Announcements` collection needed for the operations. Create with 
+
+    `python3 ../../database-collection-schemas/createAnnouncementCollection.py`.
+## Get announcements
+A simple function that parses request and return announcements (from admin) in string.
+
+A json payload looks like this:
+```json
+{ "data": "{\"getAnnouncements\":true,\"addAnnouncements\": false,\"numberOfAnnouncements\":3,\"timestamp\":1637100904,\"after\":true}" }
+```
+
+- The payload must contain `"data"` key with value in string. This string will be parsed to JSON by the function later on.
```

Comment:
Is this the only way how nested JSON objects can be parsed in Python? It would be nice, if this doesn't need to be a string.

Reply:
according to Appwrite documentation, the value for "data" key must be in string format. Parsing String to JSON on the receiver end is very easy though.